docs/faas/aws-lambda.md

```diff
@@ -54,21 +54,21 @@ and the [cloud resource conventions][cloud]. The following AWS Lambda-specific a
 [faasres]: /docs/resource/faas.md (FaaS resource conventions)
 [cloud]: /docs/resource/cloud.md (Cloud resource conventions)
 
-### AWS X-Ray Environment Span Link
+### AWS X-Ray Environment `SpanContext`
 
-If the `_X_AMZN_TRACE_ID` environment variable is set, instrumentation SHOULD try to parse an
-OpenTelemetry `Context` out of it using the [AWS X-Ray Propagator](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/context/api-propagators.md). If the
-resulting `Context` is [valid](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/trace/api.md#isvalid) then a [Span Link][] SHOULD be added to the new Span's
-[start options](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/trace/api.md#specifying-links) with an associated attribute of `source=x-ray-env` to
-indicate the source of the linked span.
-Instrumentation MUST check if the context is valid before using it because the `_X_AMZN_TRACE_ID` environment variable can
-contain an incomplete trace context which indicates X-Ray isn’t enabled. The environment variable will be set and the
-`Context` will be valid and sampled only if AWS X-Ray has been enabled for the Lambda function. A user can
-disable AWS X-Ray for the function if the X-Ray Span Link is not desired.
+If the `_X_AMZN_TRACE_ID` environment variable is set, instrumentation MUST ensure that the value from that
+environment variable is included in the `Carrier` used to extract a parent `Context` with the `Field` name
+"X-Amzn-Trace-Id" unless the `Carrier` already includes a value for that `Field`.
```

Comment:
This was originally pointed out to me by @tylerbenson but I'm adding this comment to make sure everyone is on the same page:

>unless the `Carrier` already includes a value for that `Field`.

Is there an expected case where there already exists a `X-Amzn-Trace-Id` in the header object but a different value would exist in the `_X_AMZN_TRACE_ID`? If active tracing is enabled, wouldn't you want to take the environment variable? Or does x-ray somehow reconcile this in the backend?